### PR TITLE
The judgment of increasing the version number during initialization

### PR DESCRIPTION
**Pull request by WeiLoy**

### CODE REVIEW

File: x/gov/gov_params.go [hunk 1]
```diff
@@ -245,9 +245,15 @@ func initParam() []*GovernParam {
 
 var ParamVerifierMap = make(map[string]ParamVerifier)
 
-func InitGenesisGovernParam(snapDB snapshotdb.DB) error {
+func InitGenesisGovernParam(snapDB snapshotdb.DB, genesisVersion uint32) error {
 	var paramItemList []*ParamItem
-	for _, param := range queryInitParam() {
+
+	initParamList := queryInitParam()
+	if genesisVersion >= uint32(0<<16|11<<8|0) {
```
Comment (GavinXu520) [inline x/gov/gov_params.go]: recommended using the const

File: x/xcom/common_config.go [hunk 2]
```diff
@@ -565,12 +565,14 @@ func CheckEconomicModel() error {
 		return err
 	}
 
-	if err := CheckZeroProduceNumberThreshold(ec.Slashing.ZeroProduceCumulativeTime, ec.Slashing.ZeroProduceNumberThreshold); nil != err {
-		return err
-	}
+	if genesisVersion >= uint32(0<<16|11<<8|0) {
```
Comment (GavinXu520) [inline x/xcom/common_config.go]: recommended using const

Comment (benbaley) [reply x/xcom/common_config.go]: this will be fixed in future

File: x/gov/gov_params.go (hunk 1 above)
Comment (benbaley) [reply x/gov/gov_params.go]: this will be fixed in future

**Pull request merged**